src/api/root.rs

```diff
@@ -15,8 +15,14 @@ pub async fn route(
     Extension(node): Extension<NodePool>,
 ) -> Result<impl IntoResponse, BlockfrostError> {
     let errors = vec![];
-    let mut node = node.get().await?;
-    let node_info = node.sync_progress().await?;
+
+    // XXX: Axum must not abort Ouroboros protocols in the middle, hence a separate Tokio task:
```

Comment:
What does `XXX:`  stands for?

Reply:
😅 I never checked, and always used it to denote something tricky. I.e. as a more important comment than usual comments. As in you have to watch out when changing code around it, or you’ll break it.

According to https://github.com/SAP/styleguides/issues/279#issuecomment-1192321729:

> * Probably popularized by a developer called Bill Joy in 1971, working on the Berkley Software Distribution (BSD)